scripts/checks.sh

```diff
@@ -22,6 +22,7 @@ mountpoint="/var/lib/${ENG}"
 
 low_mem_threshold=10 #%
 low_disk_threshold=10 #%
+wifi_threshold=40 #%
```

Comment:
This is 100% arbitrary. It would be good to test this more thoroughly (perhaps we can omit this test entirely for now and just begin to collect the data in the diagnostics?). cc @majorz who may have a better feeling on this.

Reply:
Maybe add a comment that it's arbitrary?

Reply:
Added a note to that effect.